openupgrade_scripts/scripts/account/15.0.1.2/upgrade_analysis_work.txt

```diff
@@ -0,0 +1,233 @@
+# NOTHING TO DO: new feature
+
+account      / account.tax              / country_id (many2one)         : NEW relation: res.country, required
+# DONE: pre-migration: filled value for country_id
+
+account      / account.tax              / is_base_affected (boolean)    : NEW hasdefault
+# NOTHING TO DO
+
+account      / account.tax.carryover.line / amount (float)                : NEW required, req_default: function, hasdefault
+account      / account.tax.carryover.line / company_id (many2one)         : NEW relation: res.company, required, req_default: function, hasdefault
+account      / account.tax.carryover.line / date (date)                   : NEW required, req_default: function, hasdefault
+account      / account.tax.carryover.line / foreign_vat_fiscal_position_id (many2one): NEW relation: account.fiscal.position
+account      / account.tax.carryover.line / name (char)                   : NEW required
+account      / account.tax.carryover.line / tax_report_line_id (many2one) : NEW relation: account.tax.report.line
+# NOTHING TO DO: new model
+
+account      / account.tax.group        / country_id (many2one)         : NEW relation: res.country
+account      / account.tax.group        / preceding_subtotal (char)     : NEW
+# NOTHING TO DO: new features
+
+account      / account.tax.report.line  / carry_over_condition_method (selection): NEW selection_keys: ['always_carry_over_and_set_to_0', 'no_negative_amount_carry_over_condition']
+account      / account.tax.report.line  / carry_over_destination_line_id (many2one): NEW relation: account.tax.report.line
+account      / account.tax.report.line  / carryover_line_ids (one2many) : NEW relation: account.tax.carryover.line
+account      / account.tax.report.line  / is_carryover_persistent (boolean): NEW hasdefault
+account      / account.tax.report.line  / is_carryover_used_in_balance (boolean): NEW
+# NOTHING TO DO: new feature
+
+account      / account.tax.template     / is_base_affected (boolean)    : NEW hasdefault
+# NOTHING TO DO: new feature
+
+account      / product.template         / account_tag_ids (many2many)   : NEW relation: account.account.tag
+# NOTHING TO DO: new feature
+
+account      / res.company              / account_fiscal_country_id (many2one): NEW relation: res.country, isfunction: function, stored
+# NOTHING TO DO: new feature
+
+account      / res.company              / account_journal_payment_credit_account_id (many2one): NEW relation: account.account
+account      / res.company              / account_journal_payment_debit_account_id (many2one): NEW relation: account.account
+# DONE post-migration: filled value from account journal
+
+account      / res.company              / account_setup_taxes_state (selection): NEW selection_keys: ['done', 'just_done', 'not_done'], hasdefault
+# NOTHING TO DO: new feature
```

Comment:
This shows a button to invite the admin user to review their taxes. I think it can be set to `done` at least for companies with any taxes.

Reply:
Do a read_group over taxes grouping by company_id and write `done` in the resulting records.